src/commands/service/pipeline.ts

```diff
@@ -149,7 +170,7 @@ export const installPipeline = async (
     repositoryName: repoName,
     repositoryUrl: repoUrl,
     yamlFileBranch: "master",
-    yamlFilePath: `packages/${serviceName}/azure-pipelines.yaml`
+    yamlFilePath: path.join(packagesDir, serviceName, "azure-pipelines.yaml") // This may not work if we're using a non-mono repository and azure-pipelines.yaml is in the root directory.
```

Comment:
Need to check this functionality based on where azure-pipelines.yaml is located...

@evanlouie @bnookala 

This probably goes into our `spk project init` rewrite and how we'll onboard services in the future.